src/components/InputTime/AsString.tsx

```diff
@@ -65,6 +66,34 @@ const AsString: React.FC<AsStringProps> = ({
   // value is the true, controlled value, patched for undefined / empty
   const value = useMemo(() => valueOrUndefined || '', [valueOrUndefined]);
 
+  const step: number | undefined = useMemo(() => {
+    let safeStep: number | undefined;
+    safeStep = typeof customStep === 'number' ? customStep : safeStep;
+
+    safeStep =
+      typeof customStep === 'string' ? parseInt(customStep, 10) : safeStep;
+
+    if (
+      !safeStep ||
+      Number.isNaN(safeStep) ||
+      safeStep < 60 * 10 ||
```

Comment:
Hmm, maybe I should change this bit.
Steps greater than a day are obviously garbage, but...

For the dropdown, we won't use steps less than 10 minutes because menu lengths.
But for actual input validation, I guess 2 minute, 5 minute steps would be valid.

I'll fix (by making a separate dropdown menu step from the `safeStep`